doc/classes/EditorPlugin.xml

```diff
@@ -820,6 +820,9 @@
 		<constant name="CONTAINER_PROJECT_SETTING_TAB_RIGHT" value="11" enum="CustomControlContainer">
 			Tab of Project Settings dialog, to the right of other tabs.
 		</constant>
+		<constant name="DOCK_SLOT_NONE" value="-1" enum="DockSlot">
+			Dock slot, none. The dock will start closed and will open as a floating window.
```

Comment:
This doesn't work that well. When you add a dock as NONE, there is no way to dock it into the editor, as closing the floating window will close the dock.

Reply:
- This will need something like https://github.com/godotengine/godot-proposals/issues/1984

I can not expose it until then.

Reply:
I assumed so. It's better to not expose an effectively broken feature.